content/user/advanced-modules/Workflow.adoc

```diff
@@ -51,7 +51,7 @@ restricted by the *Run on* property and by the Workflow's conditions.
 |================================================================
 |New Records | The Workflow will run only for new records being created.
 |Modified Records | The Workflow will run only for records that were already created and are being changed.
-|All Records | 
+|All Records a| 
```

Comment:
What's with the a? :thinking:

Reply:
This adds an AsciiDoc style to the table cell and nothing more.

Reply:
For example, without this feature:
\- For *On Save* workflows, this means the Workflow will run for both new and modified records. 
\- For *On Scheduler* Workflows, this means the Workflow will go through every record in the module, 

With this feature:
* For *On Save* workflows, this means the Workflow will run for both new and modified records. 
* For *On Scheduler* Workflows, this means the Workflow will go through every record in the module,

Reply:
Ahh I see it.
![Screenshot_2020-09-08 Workflows](https://user-images.githubusercontent.com/4041275/92457684-6d698900-f1bc-11ea-85ad-281e17698a08.png)